frontend/app/mocks/power-platform-api.server.ts

```diff
@@ -96,9 +107,37 @@ export function getPowerPlatformApiMockHandlers() {
         }),
       );
     }),
+    /**
+     * Handler for GET requests to retrieve letters details.
+     */
+    http.get('https://api.example.com/letters-types', () => {
+      const letterTypesEntities = getAllLetterTypes();
+
+      return HttpResponse.json(
+        letterTypesEntities?.map((letterType) => {
+          return {
+            nameEn: letterType.nameEn,
+            nameFr: letterType.nameFr,
+            code: letterType.code,
+            id: letterType.id,
+          };
+        }),
+      );
```

Comment:
```suggestion
      http.get('https://api.example.com/letters-types', () => {
      return HttpResponse.json(
        getAllLetterTypes().map((letterType) => ({
          nameEn: letterType.nameEn,
          nameFr: letterType.nameFr,
          code: letterType.code,
          id: letterType.id,
        })),
      );
    }),
```

Reply:
This should be `letter-types` not `letters-types`.

Reply:
Fixed: c484a60f267a6549d761d0e41ff16f7040ea61e6 and 45ef18f0a25a877da32b0caf302ed7bf2e424159